pkg/document/crdt/tree.go

```diff
@@ -433,7 +498,7 @@ func (t *Tree) Edit(from, to *TreePos, content *TreeNode, editedAt *time.Ticket)
 				}
 			}
 		} else {
-			if fromPos.Node.Parent.Value.IsRemoved() {
+			if fromPos.Node.Parent != nil && fromPos.Node.Parent.Value.IsRemoved() {
```

Comment:
May I know in which case the problem occurs?

I guess that `fromPos.node` is the root node, I'm wondering if we can just ignore it or if we need to do something else.

Reply:
As you said, we need this line of code when fromPos.Node is root of the tree. 
I think no other process is needed